tools/lint/lint.py

```diff
@@ -367,7 +367,7 @@ def check_unique_testharness_basenames(repo_root, paths):
     file_dict = defaultdict(list)
     for path in paths:
         source_file = SourceFile(repo_root, path, "/")
-        if source_file.type != "testharness":
+        if "testharness" not in source_file.possible_types:
```

Comment:
Don't we use the manifest in the lint these days? Couldn't we look up the path in the manifest and use that to tell if it's a testharness test?

Reply:
No, we don't use the manifest

Reply:
Oh. We really should :p